templates/CRM/Contact/Form/Edit/Demographics.tpl

```diff
@@ -34,17 +34,4 @@
  </div>
 </details>
 
-{literal}
-<script type="text/javascript">
-    showDeceasedDate( );
-    function showDeceasedDate( )
-    {
-        if ( cj("#is_deceased").is(':checked') ) {
-            cj("#showDeceasedDate").show( );
-        } else {
-    cj("#showDeceasedDate").hide( );
-         cj("#deceased_date").val('');
```

Comment:
We no longer need to clear the deceased date with js because we clear it properly in the forms layer. Kind of odd anyways that if you uncheck deceased, re-check it and then save you lose the date.